### PR TITLE
Update retention to be 32d,92d,0d

### PR DESCRIPTION
**Pull request by metalmatze**
As discussed in previous meetings.

/cc @brancz @kakkoyun @squat

### CODE REVIEW

File: environments/kubernetes/kube-thanos.libsonnet [hunk 1]
```diff
@@ -56,9 +56,9 @@ local rolebinding = k.rbac.v1.roleBinding;
                   args: [
                     'compact',
                     '--wait',
-                    '--retention.resolution-raw=16d',
-                    '--retention.resolution-5m=16d',
-                    '--retention.resolution-1h=16d',
+                    '--retention.resolution-raw=32d',
+                    '--retention.resolution-5m=92d',
+                    '--retention.resolution-1h=10000d',
```
Comment (brancz) [inline environments/kubernetes/kube-thanos.libsonnet]: I would change this to `0d`, which disables retention

**Comment by brancz** [comment]
small comment, otherwise lgtm

**Comment by metalmatze** [comment]
Apparently `0d` disables the retention. Are we still compacting but then keep all of those blocks forever? I'm a bit confused about that part. Thus I set it to 10000d for now :smile:

**Comment by brancz** [comment]
It means it will downsample to 1h and keep samples at 1h resolution forever

**Pull request merged**